app/Services/Manager.php

```diff
@@ -495,4 +495,27 @@ public function addUserToModel($model, $id, $user)
 
         return true;
     }
+
+    /**
+     * Fires off an event to send an email a user.
+     *
+     * @param  Gladiator\Models\User     $user
+     * @param  Gladiator\Models\Contest  $contest
+     */
+    public function sendEmail($user, $contest)
```

Comment:
also, just in case the message doesn't exist for whatever reason... maybe wrap the `$resources` and `event()` in an `if`

```php
if ($message) {
  ... // do all the sending stuff here since we found the message
}
```